bioptim/interfaces/acados_interface.py

```diff
@@ -236,7 +236,8 @@ def __update_solver(self):
         param_init = []
         for n in range(self.acados_ocp.dims.N):
             self.ocp_solver.cost_set(n, "yref", np.concatenate([data[n] for data in self.y_ref])[:, 0])
-            self.ocp_solver.cost_set(n, "W", self.W)
+            # check following line
+            # self.ocp_solver.cost_set(n, "W", self.W)
```

Comment:
Why did you comment this instead of removing?

Reply:
Because ultimately it will be written as is. I just need to check what is going on on acados side before uncommenting !

Reply:
Is this supposed to be put back during this PR?

Reply:
Nope, later